tasks/configs/copy.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  'create-artifacts-production': {
+    files: [
+      {
+        expand: true,
+        src: ['package.json', 'dist/**', 'bin/**'],
```

Comment:
Is there an easy way to make this throw an error if `dist` is empty? A simple message like 'run the build command first' might prevent some confusion.

Reply:
I looked at [`grunt-contrib-copy`](https://github.com/gruntjs/grunt-contrib-copy) but I don't see anything obvious. It's not worth spending too much time on. Maybe just add a comment to this file that you need to build first?